lib/TransportRPCServer.js

```diff
@@ -3,6 +3,12 @@
 const TransportRPC = require('./TransportRPC')
 
 class TransportRPCServer extends TransportRPC {
+  constructor (client, conf, Socket) {
+    super(client, conf)
+
+    this.Socket = Socket
```

Comment:
@prdn i still have the other commit in my reflog. 

i can easily revert to the transport-getSocket version, what do you think?